apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -372,6 +374,31 @@ export class NotificationService {
   private async sendReadyForCourtNotifications(
     theCase: Case,
   ): Promise<SendNotificationResponse> {
+    if (isIndictmentCase(theCase.type)) {
+      const email = theCase.court?.notificationEmail
+      const {
+        subject,
+        body,
+      } = formatCourtIndictmentReadyForCourtEmailNotification(
+        this.formatMessage,
+        theCase,
+        `${this.config.clientUrl}${INDICTMENTS_COURT_OVERVIEW_ROUTE}/${theCase.id}`,
+      )
+      const recipient = await this.sendEmail(
+        subject,
+        body,
+        theCase.court?.name,
+        email,
+      )
+
+      return this.recordNotification(
+        theCase.id,
+        NotificationType.READY_FOR_COURT,
+        [recipient],
+      )
+    }
```

Comment:
Consider extracting this to a separate function `sendReadyForCourtEmailNotificationToCourt`.